arrow/src/test/java/org/apache/iceberg/arrow/vectorized/ArrowReaderTest.java

```diff
@@ -263,6 +265,60 @@ public void testReadColumnFilter2() throws Exception {
         scan, NUM_ROWS_PER_MONTH, 12 * NUM_ROWS_PER_MONTH, ImmutableList.of("timestamp"));
   }
 
+  @Test
+  public void testIssue10275() throws Exception {
+    rowsWritten = Lists.newArrayList();
+    tables = new HadoopTables();
+    tableLocation = tempDir.toURI().toString();
+
+    // Define the initial table schema
```

Comment:
no need for comments on self-explanatory code

Reply:
removed